dist/streamr-client.node.js

```diff
@@ -0,0 +1,14020 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
I took a look and it seems that you could use `axios` instead of `node-fetch` and `websocket` instead of `ws` and the same code would work in both envs. I can try it a bit later

Reply:
a) Agreed. We should make sure though that the published npm package contains the `dist` folder.

b) Yeah... it should be possible, but I couldn't figure it out on the first try. A couple of constraints we need to consider:

- File size should not grow much due to including stuff that's not needed in the browser. Adding webpack has already made the file size 

- Softer thing: Using the `fetch` api is preferred since it's a sort of standard, instead of introducing yet another library to learn (`axios`).

Reply:
a) done

Reply:
@Aapzu About b): What exactly is the problem with builds for node and browser? Why is it hard to use in webpack? 

Looking at other libs, it seems fairly common to simply write commonjs modules instead of ES6 modules (use `require`s instead of `imports`). This way it works in node natively without transpiling. 

Due to using ES6 modules here I need to transpile for both node AND the browser, which leads to two builds. In `package.json` the `main` entry point (used by node when `require`d in another project) is the transpiled ES5 file in `dist`. Webpack should work normally? (Haven't tried to include this lib in a webpack project)

Reply:
Solved, I think, after reading [this](https://github.com/rollup/rollup/wiki/pkg.module). With the following in `package.json`, node uses `main` as entry point and webpack uses `module` as entry point:

```
  "main": "dist/streamr-client.node.js",
  "module": "src/index.js",
```